cli/azd/cmd/init.go

```diff
@@ -242,25 +250,28 @@ func (i *initAction) Run(ctx context.Context) (*actions.ActionResult, error) {
 			return nil, err
 		}
 
-		// Prompt to deploy to Azure
-		deploy, err := i.console.Confirm(ctx, input.ConsoleOptions{
-			Message:      "Deploy to Azure now?",
-			DefaultValue: true,
-		})
-		if err != nil {
-			return nil, err
-		}
-
-		if deploy {
-			// Call azd up
-			startTime := time.Now()
-			i.azd.SetArgs([]string{"up", "--cwd", azdCtx.ProjectDirectory()})
-			err := i.azd.ExecuteContext(ctx)
-			header = "New project initialized! Provision and deploy to Azure is completed in " +
-				ux.DurationAsText(since(startTime)) + "."
+		if i.flags.up {
+			// Prompt to deploy to Azure
+			deploy, err := i.console.Confirm(ctx, input.ConsoleOptions{
+				Message: "Template files have been initialized in your local directory. " +
+					"If you want to provision and deploy now without making changes, select Y. If not, select N.",
```

Comment:
```suggestion
                Message: "Do you want to run " + output.WithHighLightFormat("azd up") + " now?",
				Help: "Template files have been initialized in your local directory. " +
					"If you want to provision and deploy now without making changes, select Y. If not, select N.",
```

I think the original suggestion from https://github.com/Azure/azure-dev/pull/4995#discussion_r2021759722 was to have both the question and help text - would something like this work?

![image](https://github.com/user-attachments/assets/eab38495-0082-42e0-9219-69da99545929)

After entering `?`:
![image](https://github.com/user-attachments/assets/61c1a015-bb5d-4e87-a851-7b5478f8e90c)

Reply:
ok